fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/starter/LoanAccountAutoStarter.java

```diff
@@ -78,6 +79,12 @@ public RBILoanRepaymentScheduleTransactionProcessor rbiLoanRepaymentScheduleTran
         return new RBILoanRepaymentScheduleTransactionProcessor();
     }
 
+    @Bean
+    @ConditionalOnProperty("fineract.loan.transactionprocessor.due-date-respective.enabled")
```

Comment:
Would be better to have the property mapped to the FineractProperties class and then using a PropertiesCondition for this. That way it's going to be type-safe and the property name is not duplicated.